lib/crowbar/client/request/backup/download.rb

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright 2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  module Client
+    module Request
+      module Backup
+        class Download < Base
+          def headers
+            {
```

Comment:
```ruby
super.easy_merge!(
  "Content-Type" => "application/octet-stream",
  "Accept" => "application/octet-stream"
)
```

Doing it the same as for the content to be able to extend the base in the future.